odl/test/operator/oputils_test.py

```diff
@@ -171,7 +171,8 @@ def test_power_method_opnorm_nonsymm():
     true_opnorm = 6
 
     # Start vector (1, 1) is close to the wrong eigenvector
-    opnorm_est = power_method_opnorm(op, maxiter=50)
+    xstart = odl.rn(2).element([1, 1])
+    opnorm_est = power_method_opnorm(op, xstart=xstart, maxiter=50)
```

Comment:
I noticed that the order of the keyword arguments is different in lines 175 and 180. Since the order in the definition of `power_method_opnorm` is `op`, `xstart`, `maxiter`, maybe line 180 should be adapted to this. Otherwise, this looks good to me.